server/runtime/src/main/java/org/infinispan/server/security/KeyStoreUtils.java

```diff
@@ -74,42 +62,6 @@ public static void generateSelfSignedCertificate(String keyStoreFileName, String
    }
 
    public static KeyStore loadKeyStore(final Supplier<Provider[]> providers, final String providerName, FileInputStream is, String filename, char[] password) throws IOException, GeneralSecurityException {
-      try {
-         return KeyStoreUtil.loadKeyStore(providers, providerName, is, filename, password);
-      } catch (KeyStoreException e) {
-         return loadPemAsKeyStore(filename, password);
-      }
-   }
-
-   public static KeyStore loadPemAsKeyStore(String filename, char[] password) throws IOException, KeyStoreException, CertificateException, NoSuchAlgorithmException {
-      KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
-      keyStore.load(null);
-      // try to load it as a PEM
-      PrivateKey pk = null;
-      List<Certificate> certificates = new ArrayList<>();
-      byte[] bytes = Files.readAllBytes(Paths.get(filename));
-      for (Iterator<PemEntry<?>> it = Pem.parsePemContent(CodePointIterator.ofUtf8Bytes(bytes)); it.hasNext(); ) {
-         Object entry = it.next().getEntry();
-         if (entry instanceof PrivateKey) {
-            // Private key
-            pk = (PrivateKey) entry;
-         } else if (entry instanceof Certificate) {
-            // Certificate
-            Certificate certificate = (Certificate) entry;
-            certificates.add(certificate);
-         }
-      }
-      if (pk != null) {
-         // A keystore
-         keyStore.setKeyEntry("key", pk, password, certificates.toArray(new Certificate[0]));
-      } else {
-         // A truststore
-         int i = 1;
-         for(Certificate certificate : certificates) {
-            keyStore.setCertificateEntry(Integer.toString(i++), certificate);
-         }
-      }
-
-      return keyStore;
+      return KeyStoreUtil.loadKeyStore(providers, providerName, is, filename, password);
```

Comment:
This method is only called in `KeyStoreConfiguration`, we scan just call `KeyStoreUtil` directly and remove this method.

Reply:
Fixed